forge-gui/res/cardsfolder/upcoming/lost_in_the_maze.txt

```diff
@@ -0,0 +1,12 @@
+Name:Lost in the Maze
+ManaCost:X U U 
+Types:Enchantment
+K:Flash
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigTap | TriggerDescription$ When CARDNAME enters the battlefield, tap X target creatures. Put a stun counter on each of those creatures you don't control. (If a permanent with a stun counter would become untapped, remove one from it instead.)
+SVar:TrigTap:DB$ Tap | TargetMin$ X | TargetMax$ X | ValidTgts$ Creature | SubAbility$ DBCounter | TgtPrompt$ Select target creatures
+SVar:DBCounter:DB$ PutCounter | Defined$ Targeted.YouDontCtrl | CounterType$ Stun | CounterNum$ 1
+S:Mode$ Continuous | Affected$ Creature.YouCtrl+tapped | AddKeyword$ Hexproof | Description$ Tapped creatures you control have hexproof.
+SVar:X:Count$xPaid
+DeckHas:Ability$Counters
+SVar:PlayMain1:TRUE
```

Comment:
Not sure about this on a card with Flash... does this tell the AI not to use this as, say, a combat trick?